CheckDrive.Api/CheckDrive.Api/Controllers/Authorization/AuthController.cs

```diff
@@ -0,0 +1,40 @@
+﻿using CheckDrive.Application.Constants;
+using CheckDrive.Application.DTOs.Identity;
+using CheckDrive.Application.Interfaces.Authorization;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace CheckDrive.Api.Controllers.Authorization;
+[Route("api/auth")]
+[ApiController]
+public class AuthController(IAuthService authService) : ControllerBase
+{
+    private readonly IAuthService _authService = authService
+        ?? throw new ArgumentNullException(nameof(authService));
+
+    [HttpPost("register/employee")]
+    [Authorize(Roles = $"{Roles.Manager},{Roles.Administrator}")]
+    public async Task<IActionResult> Register(RegisterDto registerUser)
+    {
+        // Register the new user with a position (role)
```

Comment:
No need for comments.

https://www.linkedin.com/pulse/role-comments-code-vitalii-serdiuk/